src/main/java/cloud/prefab/client/value/Value.java

```diff
@@ -0,0 +1,9 @@
+package cloud.prefab.client.value;
+
+public interface Value<T> {
```

Comment:
I wonder if we can get by with `Supplier<T>` which is in the JDK, and therefore prevents prefab types from infecting too much of your codebase

Reply:
We lose the `or` and `orNull` but I'm ok with that. Sounds like a good place to start.

Reply:
@jhaber if we do `public interface Value<T> extends Supplier<T> ` does that nicely have our cake and eat it too? Then people could choose. 🎱

Reply:
Sounds good